nodes/test/nodes_tests/test_plugins.py

```diff
@@ -169,7 +168,7 @@ def setUp(self):
         self.alias = (self.parentfs, self.parentfs)
         Consumer.get_collection().remove()
         Bind.get_collection().remove()
-        RepoDistributor.get_collection().remove()
+        model.Distributor.drop_collection()
```

Comment:
Does dropping the collection also remove indicies? If so, then sticking with a remove call would be better.